client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -15,7 +16,7 @@ export const AppsViewLayout: React.FC = () => {
   return (
     <>
       {htmlApp ? (
-        <div id="appDetail-wrapper" className="has-external-app">
+        <div className={styles['center-overflow']}>
```

Comment:
- ✅ Removing the unused `id` is great.
- ✅ Replacing global class with local class is great.
- 🤔 Replacing specific class (`has-external-app`) with a one-off utility class (`center-overflow`) isolated to a single module seems like a "Git-R-Done", but I do **not** know the codebase as well as you do, and I **do** know the timeline, and I